### ISSUE TITLE
public/index.html add lang attribute to <html>

### ISSUE THREAD

**Issue by kerryran** [WVU-CS230-2022-08-Group10/Logique-a-la-Crabe-2.0#2]
Index.html file is missing a language attribute to the <html> tag. It is necessary for the user experience and accessibility so things like screen readers know what language the document is in and can read it properly. 
To recreate
- Add a <html> tag
- Do not include lang="en" inside the tag

Kerrigan Ranshaw
